src/content/plugins/source-map-dev-tool-plugin.md

```diff
@@ -45,3 +47,14 @@ new webpack.SourceMapDevToolPlugin({
   exclude: ['vendor.js']
 })
 ```
+
+### Host sourcemaps externally
+
+Set a URL for sourcemaps. Useful for hosting them on a host that requires authorization.
```

Comment:
Split "sourcemaps" into two words ("source maps") here as well. Just for consistency with the rest of our docs.